index.js

```diff
@@ -7,13 +7,26 @@ const YAML = require('js-yaml')
 const TOML = { parse: require('toml-j0.4').parse, stringify: require('tomlify-j0.4') }
 const INI = require('ini')
 
+/**
+ * Removes '.' in file extensions
+ * @param  {string} type input file extension
+ * @return {string}      file extension without '.'
+ */
+const validType = (type) => {
+    if (type.charAt(0) === '.') {
+        type = type.substr(1)
+    }
+    return type
+}
+
 const parse = (string, type) => {
     if (!string) throw new Error('missing string')
     if (!type) throw new Error('missing type')
+    type = validType(type)
     switch (type) {
```

Comment:
Were there in the future, to be more code in between, wouldn't it be more readable by stating per line an action instead of having as many actions in one line as possible, minifiers are there for that purpose